ckanext/canada/tables/contracts.yaml

```diff
@@ -1138,9 +1148,6 @@ iii. Quand un contrat porte sur plus d’un article économique, il est recomman
       "32":
         en: Subscriptions to newspapers, magazines, or other periodicals (for contracts that are only covered by the CFTA)
         fr: Abonnements à des journaux, des magazines ou d’autres périodiques (pour les marchés qui sont visés par l’ALEC seulement)
-      "33":
-        en: Substantial duplication of costs to the procuring entity (for contracts that are only covered by the CFTA)
-        fr: Multiplication importante des coûts pour l’entité acheteuse (pour les marchés qui sont visés par l’ALEC seulement)
```

Comment:
note to self: removing this option doesn't require data migration because it was added in contracts-changes branch (not deployed)